.github/workflows/docker.yml

```
@@ -0,0 +1,122 @@
+# ABOUTME: GitHub Actions workflow for Docker image build and testing
+# ABOUTME: Builds and tests Docker containers for both development and production
+
+name: Docker
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)